rbac/management/role/model.py

```diff
@@ -174,10 +174,9 @@ def unassign_group(self, group_uuid) -> Optional[Relationship]:
         """
         relationship = None
         while True:
-            previous_relationship = self.pop_group_from_bindings(group_uuid)
-            if previous_relationship is None:
+            relationship = self.pop_group_from_bindings(group_uuid)
+            if relationship is not None:
                 break
-            relationship = previous_relationship
         return relationship
```

Comment:
I don't think this is right? We want to remove all of them, but this will only remove one, because it stops once the relationship is not None. If there is more than one, does this work?